src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusQueueResource.cs

```diff
@@ -97,7 +97,24 @@ public string QueueName
 
     // ensure Azure Functions projects can WithReference a ServiceBus queue
     void IResourceWithAzureFunctionsConfig.ApplyAzureFunctionsConfiguration(IDictionary<string, object> target, string connectionName)
-        => ((IResourceWithAzureFunctionsConfig)Parent).ApplyAzureFunctionsConfiguration(target, connectionName);
+    {
+        if (Parent.IsEmulator)
```

Comment:
Thoughts on consolidating this code into the parent SB Resource as well? Just like we did for conneciton string?